cranetoolbox/importTools/transform.py

```diff
@@ -221,30 +234,54 @@ def parse_tweet(tweet: str) -> dict:
     return json.loads(tweet)
 
 
-def lighten_tweet(tweet: dict) -> (str, str, str):
+def lighten_tweet(tweet: dict, text_name, id_name, date_name) -> (
+str, str, str):
     """Lighten a tweet by returning only the fields required for analysis.
 
     :param tweet: A parsed JSON tweet
     :type tweet: dict
+    :param text_name : A string to different text field name
```

Comment:
You don't need to specify that it's a string since the type is on the following line.
Could you replace those param descriptions by User-defined name for the "text"/"id"/"created_at" field, please?

Reply:
It is true! okay, i will replace.